collectors/metrics/cmd/metrics-collector/main.go

```diff
@@ -554,68 +537,241 @@ func initConfig(o *Options) (error, *forwarder.Config) {
 	transformer.With(metricfamily.TransformerFunc(metricfamily.PackMetrics))
 	transformer.With(metricfamily.TransformerFunc(metricfamily.SortMetrics))
 
-	isHypershift, err := metricfamily.CheckCRDExist(o.Logger)
-	if err != nil {
-		return err, nil
+	// TODO(saswatamcode): Kill this feature.
```

Comment:
I agree, most of what is done in the metrics collector is already setup in the endpoint-operator when generating the ServiceMonitors. The only missing labels are `managementcluster` and `managementclusterID`. They can be easily added.

Reply:
Yup, we can think of a way to add it as a relabel config on Prom, using MCO. But probably in later PRs.